code/parachain/runtime/primitives/src/topology.rs

```diff
@@ -154,8 +155,25 @@ impl Composable {
 				ratio: Some(rational!(1 / 1)),
 			},
 		);
+		let dot = (
+			6,
+			Some(ForeignAssetId::Xcm(XcmAssetLocation::new(MultiLocation::new(1, Here)))),
+			AssetInfo {
+				name: Some(
+					BiBoundedAssetName::from_vec(b"Polkadot".to_vec())
+						.expect("String is within bounds"),
+				),
+				symbol: Some(
+					BiBoundedAssetSymbol::from_vec(b"DOT".to_vec())
+						.expect("String is within bounds"),
+				),
+				decimals: Some(10),
+				existential_deposit: 1_000_000,
+				ratio: Some(rational!(2143 / 100_000_000)),
```

Comment:
this is also different from migrated dot on composable 
existential_deposit: 214_300_000,
ratio: Some(rational!(2143 / 1_000_000))